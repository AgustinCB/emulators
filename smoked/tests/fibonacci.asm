.data
INTEGER 1
INTEGER 2
INTEGER 40
FUNCTION 20 1
.code
CONSTANT 2
CONSTANT 3
CALL
RETURN
GET_LOCAL 0
CONSTANT 1
LESS
JMP_IF_FALSE 10
GET_LOCAL 0
RETURN
GET_LOCAL 0
CONSTANT 0
MINUS
CONSTANT 3
CALL
GET_LOCAL 0
CONSTANT 1
MINUS
CONSTANT 3
CALL
PLUS
RETURN
