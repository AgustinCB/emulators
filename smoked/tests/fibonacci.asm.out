0 - Integer(8)
